pages/services/elastic/2.5.0-6.3.2/how-to-guides/index.md

```diff
@@ -29,37 +29,39 @@ edgelb-pool  v1.3.0   True      True       EdgeLB Pool on DC/OS
 
 If it does, you can skip the `dcos package repo add` commands below.
 
-*Otherwise*, if you see a `No packages found` message you'll need to add a couple of package repositories to your cluster. These are for EdgeLB version 1.3.0, which is the latest at the time of writing. Take a look at the [EdgeLB service documentation](https://docs.mesosphere.com/services/edge-lb/) to check if that's the latest version.
+*Otherwise*, if you see a `No packages found` message you'll need to add a couple of package repositories to your cluster. For information about the current Edge-LB version support and compatibility, see the [Edge-LB documentation](/services/edge-lb/) and the [Certified packages and DC/OS versions](/version-policy/#certified-packages-and-dcos-versions/) to compatibility matrix.
 
 ```bash
-dcos package repo add edgelb https://downloads.mesosphere.com/edgelb/v1.3.0/assets/stub-universe-edgelb.json
-dcos package repo add edgelb-pool https://downloads.mesosphere.com/edgelb-pool/v1.3.0/assets/stub-universe-edgelb-pool.json
+dcos package repo add edgelb https://<insert download link>/stub-universe-edgelb.json
+
+dcos package repo add edgelb-pool https://<insert download link>/stub-universe-edgelb-pool.json
```

Comment:
The [Edge-LB installation documentation](https://docs.mesosphere.com/services/edge-lb/1.3/installing/) *does not have* URLs for repositories containing the Edge-LB packages. Could we please leave the actual URLs until they have it in their documentation.

If not, customers will be *unable* to follow these instructions step by step.